<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" ignored="0" total="2" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Surefire suite" duration-ms="28226" started-at="2019-01-21T01:32:07Z" finished-at="2019-01-21T01:32:36Z">
    <groups>
    </groups>
    <test name="Surefire test" duration-ms="28226" started-at="2019-01-21T01:32:07Z" finished-at="2019-01-21T01:32:36Z">
      <class name="LoginTest">
        <test-method status="PASS" signature="setUp(java.lang.String)[pri:0, instance:LoginTest@13969fbe]" name="setUp" is-config="true" duration-ms="6714" started-at="2019-01-21T01:32:08Z" finished-at="2019-01-21T01:32:14Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="invalidLoginTest_InvalidUserNameInvalidPassword()[pri:0, instance:LoginTest@13969fbe]" name="invalidLoginTest_InvalidUserNameInvalidPassword" duration-ms="6628" started-at="2019-01-21T01:32:14Z" finished-at="2019-01-21T01:32:21Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [E-posta adresiniz veya şifreniz hatalı] but found [By.xpath: //*[@id="loginForm"]/div[2]/div/div ]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [E-posta adresiniz veya şifreniz hatalı] but found [By.xpath: //*[@id="loginForm"]/div[2]/div/div ]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:442)
at pages.allpages.LoginPage.verifyLoginPassword(LoginPage.java:41)
at LoginTest.invalidLoginTest_InvalidUserNameInvalidPassword(LoginTest.java:26)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:497)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:132)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:193)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:94)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:147)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- invalidLoginTest_InvalidUserNameInvalidPassword -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:LoginTest@13969fbe]" name="tearDown" is-config="true" duration-ms="733" started-at="2019-01-21T01:32:21Z" finished-at="2019-01-21T01:32:22Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp(java.lang.String)[pri:0, instance:LoginTest@13969fbe]" name="setUp" is-config="true" duration-ms="6153" started-at="2019-01-21T01:32:22Z" finished-at="2019-01-21T01:32:28Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="invalidLoginTest_EmptyUserEmptyPassword()[pri:1, instance:LoginTest@13969fbe]" name="invalidLoginTest_EmptyUserEmptyPassword" duration-ms="5242" started-at="2019-01-21T01:32:28Z" finished-at="2019-01-21T01:32:33Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Lütfen e-posta adresinizi girin.] but found [By.xpath: //*[@id="loginForm"]/div[1]/div/div]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Lütfen e-posta adresinizi girin.] but found [By.xpath: //*[@id="loginForm"]/div[1]/div/div]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:442)
at pages.allpages.LoginPage.verifyLoginUserName(LoginPage.java:35)
at LoginTest.invalidLoginTest_EmptyUserEmptyPassword(LoginTest.java:43)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:497)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:132)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:193)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:94)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:147)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- invalidLoginTest_EmptyUserEmptyPassword -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:LoginTest@13969fbe]" name="tearDown" is-config="true" duration-ms="2713" started-at="2019-01-21T01:32:33Z" finished-at="2019-01-21T01:32:36Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- LoginTest -->
    </test> <!-- Surefire test -->
  </suite> <!-- Surefire suite -->
</testng-results>
